### PR TITLE
New package: ChenQingYu.BF version 1.0.0

### PR DESCRIPTION
**Pull request by chen-qingyu**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132221)

### CODE REVIEW

**Comment by stephengillie** [comment]
I believe the programming language name in the description might be triggering the [Policy 2.5](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#25-offensive-content) check.

File: manifests/c/ChenQingYu/BF/1.0.0/ChenQingYu.BF.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageLocale: en-US
+Publisher: Chen QingYu
+PublisherUrl: https://github.com/chen-qingyu
+PublisherSupportUrl: https://github.com/chen-qingyu/bf/issues
+Author: Chen QingYu
+PackageName: Brainfuck interpreter
+PackageUrl: https://github.com/chen-qingyu/bf
+License: GPL-3.0
+LicenseUrl: https://github.com/chen-qingyu/bf/blob/main/license
+ShortDescription: A simple Brainfuck interpreter implemented in C.
```
Comment (stephengillie) [inline manifests/c/ChenQingYu/BF/1.0.0/ChenQingYu.BF.locale.en-US.yaml:15]:
```suggestion
ShortDescription: A simple Brainf**k interpreter implemented in C.
```
This appears to be a "radio-safe" common spelling for the language. Is this change acceptable?

Comment (chen-qingyu) [reply manifests/c/ChenQingYu/BF/1.0.0/ChenQingYu.BF.locale.en-US.yaml]: Of course, thank you.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**